src/sparsezoo/model/model.py

```diff
@@ -587,12 +587,12 @@ def _directory_from_files(
         regex: Optional[bool] = False,
         allow_multiple_outputs: Optional[bool] = False,
         **kwargs: object,
-    ) -> Union[Directory, None]:
+    ) -> Union[List[Union[Directory, Any, None]], List[Directory], None]:
```

Comment:
Fix typing

Reply:
`Any` doesn't cover the `None` case?

Reply:
Hmm, you are correct it should

Reply:
But I believe it's nicer to make it explicit, will remove if it's a hard blocker for you